### PR TITLE
feat: block request by url

### PR DESCRIPTION
**Pull request by tofuliang**
just like this.
image

### CODE REVIEW

File: src/ext-list.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ export async function updateList(exts = extensions, el = document.getElementById
       </summary>
       <div class='requests'>
       ${Object.keys(ext.reqUrls).map(url => `
+        <div class='item-block pointer ${ext.reqUrls[url].isBlocked ? ' blocked' : ''}' id='block-ext-url-${url}|${ext.id}' title='${ext.reqUrls[url].isBlocked ? 'Unblock' : 'Block'} network requests for this url'></div>
```
Comment (dnakov) [inline src/ext-list.js:41]: I think there were some restrictions on what characters are allowed in the HTML `id` attribute, not sure if some URL's would work as `id`

Comment (tofuliang) [reply src/ext-list.js]: please check https://github.com/dnakov/little-rat/pull/18/commits/215586cedea8209c97be636366a890824be11b2e this commit.

Comment (dnakov) [reply src/ext-list.js]: That looks good, I think the URL is already encoded, so there won't be spaces in it.

**Comment by tofuliang** [comment]
> @tofuliang thanks again for the PR! Can you remove the build and .crx from the PR? I'm distributing it as just the ZIP from github (.crx had some issues) and publishing manually to store via the build scripts.

I use this files for auto update, so I will open a new PR, hope you don't mind.

**Pull request closed without merging**